### PR TITLE
Allow users to enable swap on their app servers

### PR DESCRIPTION
**Pull request by jvanbaarsen**
When you are at the limits of the resources on the app server,
You want to be able to enable Swap so you can squeesh out the latest
bits of resources out of the server.

By allowing users to easily enable and disable swap in the interface,
we support this feature

Fixes: #83

### CODE REVIEW

File: app/jobs/disable_swap_job.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  private
+
+  def upload_swap_script
+    SshExecution.new(server).scp(from: Rails.root.join("scripts", "disable_swap.sh"),
+                                 to: "/root/disable_swap.sh",
+                                 direction: :upload)
+  end
+
+  def disable_swap
+    SshExecution.new(server).execute(command: "sh /root/disable_swap.sh")
```
Comment (brambokdam) [inline app/jobs/disable_swap_job.rb]: We could make a variable of the path `/root/disable_swap.sh`.

Comment (jvanbaarsen) [inline app/jobs/disable_swap_job.rb]: What would the added benefit be? (Not opposed to the idea, just curious 😄 )

**Comment by joshuajansen** [comment]
Nice feature, nice pull @jvanbaarsen 😄

File: app/jobs/disable_swap_job.rb (hunk 1 above)
Comment (brambokdam) [inline app/jobs/disable_swap_job.rb]: Seems a bit cleaner as we use the same path 3 times. :)

**Comment by brambokdam** [comment]
One suggestion other then that :shipit:

File: app/jobs/disable_swap_job.rb (hunk 1 above)
Comment (jvanbaarsen) [inline app/jobs/disable_swap_job.rb]: 👍

**Pull request merged**